nifi-nar-bundles/nifi-elasticsearch-5-bundle/nifi-elasticsearch-5-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,252 @@
+      the program and the nonexistence of any infringement or violation of
+      any right of any third party.
+
+      Each user of the program will agree and understand, and be deemed to
+      have agreed and understood, that there is no warranty whatsoever for
+      the program and, accordingly, the entire risk arising from or
+      otherwise connected with the program is assumed by the user.
+
+      Therefore, neither ICOT, the copyright holder, or any other
+      organization that participated in or was otherwise related to the
+      development of the program and their respective officials, directors,
+      officers and other employees shall be held liable for any and all
+      damages, including, without limitation, general, special, incidental
+      and consequential damages, arising out of or otherwise in connection
+      with the use or inability to use the program or any product, material
+      or result produced or otherwise obtained by using the program,
+      regardless of whether they have been advised of, or otherwise had
+      knowledge of, the possibility of such damages at any time during the
+      project or thereafter.  Each user will be deemed to have agreed to the
+      foregoing by his or her commencement of use of the program.  The term
+      "use" as used herein includes, but is not limited to, the use,
+      modification, copying and distribution of the program and the
+      production of secondary products from the program.
+
+      In the case where the program, whether in its original form or
+      modified, was distributed or delivered to or received by a user from
+      any person, organization or entity other than ICOT, unless it makes or
+      grants independently of ICOT any specific warranty to the user in
+      writing, such person, organization or entity, will also be exempted
+      from and not be held liable to the user for any such damages as noted
+      above as far as the program is concerned.
+
+  (ASLv2) Carrotsearch HPPC
+      The following NOTICE information applies:
+      HPPC borrowed code, ideas or both from:
+
+       * Apache Lucene, http://lucene.apache.org/
+         (Apache license)
+       * Fastutil, http://fastutil.di.unimi.it/
+         (Apache license)
+       * Koloboke, https://github.com/OpenHFT/Koloboke
+         (Apache license)
+
+  (ASLv2) Joda Time
+      The following NOTICE information applies:
+        This product includes software developed by
+        Joda.org (http://www.joda.org/).
+
+  (ASLv2) The Netty Project
+      The following NOTICE information applies:
+        The Netty Project
+        Copyright 2011 The Netty Project
+
+  (ASLv2) t-digest
+      The following NOTICE information applies:
+      The code for the t-digest was originally authored by Ted Dunning
+      A number of small but very helpful changes have been contributed by Adrien Grand (https://github.com/jpountz)
+
+  (ASLv2) Apache Commons-CLI
```

Comment:
I went through the dependencies and didn't find this one. I believe it can be removed from the NOTICE.